impeller/renderer/backend/vulkan/render_pass_builder_vk.h

```diff
@@ -42,6 +42,15 @@ class RenderPassBuilderVK {
 
   vk::UniqueRenderPass Build(const vk::Device& device) const;
 
+  // Visible for testing.
+  const std::map<size_t, vk::AttachmentDescription>& GetColors() const;
```

Comment:
`GetColorAttachments` perhaps? To better match the `Set` calls.

Reply:
Done